README.md

```diff
@@ -1,4 +1,4 @@
-bootstrap-slider [![Build Status](https://travis-ci.org/seiyria/bootstrap-slider.png?branch=master)](https://travis-ci.org/seiyria/bootstrap-slider) [![Code Climate](https://codeclimate.com/github/seiyria/bootstrap-slider/badges/gpa.svg)](https://codeclimate.com/github/seiyria/bootstrap-slider)
```

Comment:
This change should not be part of your pull request. This change was to remove `codeclimate` checks from Github. I don't know how it ended up in your pull request. I suggest pulling down the latest version of `master` using

But first you need to upstream as a remote.

```bash
git remote add upstream git@github.com:seiyria/bootstrap-slider.git
```

```bash
# get latest changes from upstream's master branch
git fetch upstream master
# checkout local branch
git checkout master
# should do a fast-forward
git merge upstream/master
```